utils/cliutils/commandsflags.go

```diff
@@ -164,8 +164,8 @@ const (
 	specVars = "spec-vars"
 
 	// Build info flags
-	buildName   = "build-name"
-	buildNumber = "build-number"
+	BuildName   = "build-name"
+	BuildNumber = "build-number"
```

Comment:
Why did we make these variables public? They are used only in this file?

Reply:
You have asked me to use BuildName and BuildNumber and not to add new ones to the command - so had to turn the current into public

Reply:
Got it. I didn't notice those flags are used from within a different package,